assets/sass/components/key-metrics/_googlesitekit-km-selection-panel.scss

```diff
@@ -91,16 +91,24 @@
 
 		.googlesitekit-error-text p {
 			margin: 0;
+
+			@media (max-width: $bp-xsmallOnly) {
+				margin: 1em 0;
+			}
 		}
 	}
 
 	.googlesitekit-km-selection-panel-footer__content {
-		align-items: center;
-		column-gap: $grid-gap-desktop;
-		display: flex;
-		flex-wrap: wrap;
-		justify-content: space-between;
-		margin-top: $grid-gap-desktop / 2;
+		margin-top: 0;
```

Comment:
I don't think this is needed as `.googlesitekit-km-selection-panel-footer__content` doesn't have any margin by default.